src/configs/heroku.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::ModuleConfig;
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+use std::collections::HashMap;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct HerokuConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+    pub app_aliases: HashMap<String, &'a str>,
```

Comment:
in #3569 I also see the kubernetes-module uses regexes for pattern replacements, which could be also interesting here. ( or for a later PR)

Reply:
I added the regex pattenr replacements now